00-RELEASENOTES

```diff
@@ -1,16 +1,213 @@
-Hello! This file is just a placeholder, since this is the "unstable" branch
-of Valkey, the place where all the development happens.
+Valkey 9.0 release notes
+========================
 
-There is no release notes for this branch, it gets forked into another branch
-every time there is a partial feature freeze in order to eventually create
-a new stable release.
+Upgrade urgency levels:
 
-Usually "unstable" is stable enough for you to use it in development environments
-however you should never use it in production environments. It is possible
-to download the latest stable release here:
+| Level    | Meaning                                                             |
+|----------|---------------------------------------------------------------------|
+| LOW      | No need to upgrade unless there are new features you want to use.   |
+| MODERATE | Program an upgrade of the server, but it's not urgent.              |
+| HIGH     | There is a critical bug that may affect a subset of users. Upgrade! |
+| CRITICAL | There is a critical bug affecting MOST USERS. Upgrade ASAP.         |
+| SECURITY | There are security fixes in the release.                            |
 
-    https://valkey.io/download/
+Valkey 9.0.0-rc1 - August 14, 2025
+----------------------------------
 
-More information is available at https://valkey.io
+Upgrade urgency LOW: This is the first release candidate of Valkey 9.0.0, with
+performance improvements, atomic slot migrations, hash field expiration, and numbered databases in cluster mode,
+as well as a whole host of new items as listed below.
 
-Happy hacking!
+### New Features
+* Introduce atomic slot migration by @murphyjacob4 (#1949)
+* Introduce HASH items expiration by @ranshid (#2089)
+* support negative filtering for client command filters by @soloestoy (#2378)
+* Add support for automatic client authentication via TLS certificate fields by @omanges (#1920)
+* Add SAFE option to SHUTDOWN to reject shutdown in unsafe situations by @enjoy-binbin (#2195)
+* Support BYPOLYGON option for GEOSEARCH by @KarthikSubbarao (#1809)
+* Add DELIFEQ command by @LinusU (#1975)
+* Add multi-database support to cluster mode by @xbasel (#1671)
+* Introduce MPTCP for replica by @pizhenwei (#1961)
+* Add --hotkeys-count option for valkey-cli by @hwware (#1933)
+* Add sentinel_total_tilt to sentinel INFO sentinel by @carlosfu (#1904)
+* Introduce MPTCP by @pizhenwei (#1811)
+* Allow dynamic modification of io-threads num by @ayush933 (#2033)
+* Client Commands Extended Filtering by @sarthakaggarwal97 (#1466)
+
+### New Configs
+* Auto-failover on shutdown unified config by @zuiderkwast (#2292)
+
+### New info fields and Logging
```

Comment:
This bundling is a bit odd. "Logging and Tooling Improvements" (used in 8.1 release) sounds better to me.